.github/workflows/rebuild_slides.R

```diff
@@ -3,6 +3,8 @@ setwd(here::here())
 
 icons::download_fontawesome()
 
+install.packages("RSQLite")
```

Comment:
I think it should be sufficient to add this package to Suggests in DESCRIPTION, rather than installing directly in this script... this script is a bit of a kludge (e.g., having to run the icons::download_fontawesome() everytime as a step)